cornflow-server/cornflow/config.py

```diff
@@ -59,11 +59,9 @@ class DefaultConfig(object):
     LDAP_PROTOCOL_VERSION = int(os.getenv("LDAP_PROTOCOL_VERSION", 3))
     LDAP_USE_TLS = os.getenv("LDAP_USE_TLS", "False")
 
-    # OpenID login -> Default Azure
-    OID_PROVIDER = os.getenv("OID_PROVIDER", 0)
-    OID_CLIENT_ID = os.getenv("OID_CLIENT_ID")
-    OID_TENANT_ID = os.getenv("OID_TENANT_ID")
-    OID_ISSUER = os.getenv("OID_ISSUER")
+    # OpenID Connect configuration
+    PROVIDER_URL = os.getenv("PROVIDER_URL")
+    EXPECTED_AUDIENCE = os.getenv("EXPECTED_AUDIENCE")
```

Comment:
I would prefer if they are called `OID_PROVIDER` and `OID_EXPECTED_AUDIENCE`

Reply:
done